applications/yolov3/README.md

```diff
@@ -1,9 +1,5 @@
 # Introduction 
 The original keras model was coming from: <https://github.com/qqwweee/keras-yolo3>, clone the project and follow the 'Quick Start' to get the pre-trained model.
 
-# Convert
-```
-export PYTHONPATH=$(the keras-yolo3 path) 
-python yolov3.py -c  # convert the model into the ONNX
-python yolov3.py data/test.jpg # run a detect example
-```
+We have converted yolov3 model successfully and uploaded to the model zoo. We have some code that are not ready to check-in here.
+So please use our pretrained yolov3 onnx model in the model zoo [here](https://github.com/onnx/models/tree/master/yolov3)
```

Comment:
Wording suggestion:

Currently, conversion code for this model is not ready to be checked in, so please use our pretrained ...